client/components/PhotoCarousel.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PhotoEntry from './PhotoEntry.jsx';
+
+class PhotoCarousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      modalView: false,
+      thumbnails: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch(`http://localhost:3333/api/photos/${this.props.id}`)
```

Comment:
I am not 100% sure, should this fetch be in a method by it's self? I am thinking down the road, you might want to do something else with the life cycle method componentDidMount. Comments?

Reply:
I don't return in the fetch promise chain, so I can actually just do more afterward if needed.